frontend/styles/components/charities/CampaignCharityDialogStyles.ts

```diff
@@ -1,5 +1,14 @@
 import { SxProps } from '@mui/material';
 
+export const dialogSx: SxProps = {
```

Comment:
Does `<Dialog PaperProps={{sx: sthSx}} ... />` work? I've tried it before

https://stackoverflow.com/questions/50649408/how-to-use-material-ui-dialog-paperprops